compact/tracker.xml

```diff
@@ -133,16 +133,17 @@
   </documentation>
   <plugins>
     <plugin name="epic_FileLoader">
-      <arg value="cache:$DETECTOR_PATH:/opt/detector"/>
+      <arg value="cache:$DETECTOR_PATH"/>
       <arg value="file:calibrations/materials-map.cbor"/>
       <arg value="url:https://eicweb.phy.anl.gov/EIC/detectors/athena/uploads/56b64d544442a93904488e7292aa509d/material-maps.cbor"/>
     </plugin>
   </plugins>
 
-  <include ref="tracker/vertex_barrel.xml"/>
-  <include ref="tracker/silicon_barrel.xml"/>
-  <include ref="tracker/mpgd_barrel.xml"/>
-  <include ref="tracker/silicon_disks.xml"/>
-  <include ref="tracker/support_service_assembly.xml"/>
+	     <include ref="tracker/vertex_barrel.xml"/>
+	     <include ref="tracker/silicon_barrel.xml"/>
+	     <include ref="tracker/mpgd_barrel.xml"/>
+	     <include ref="tracker/mpgd_dirc.xml"/>
+	     <include ref="tracker/silicon_disks.xml"/>
+	     <include ref="tracker/support_service_assembly.xml"/>
```

Comment:
No tabs please :-)

Reply:
Ok, let me remove the tabs and I will update

Reply:
fixed